.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,35 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ next, beta, master ]
+  pull_request:
+    branches: [ next, beta, master ]
+  schedule:
+    - cron: '39 21 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
+    - name: Initialize CodeQL
+      uses: github/codeql-action/init@v1
+      with:
+        languages: ${{ matrix.language }}
+
+    - name: Autobuild
+      uses: github/codeql-action/autobuild@v1
```

Comment:
I think this is only for compiled languages, which wouldn't apply to our package. https://github.com/github/codeql-action#usage

> Autobuild attempts to build any compiled languages (C/C++, C#, or Java).
> If this step fails, then you should remove it and run the build manually (see below).

I'd think we'd want to run a `yarn install && yarn build` so the analyze happens against compiled as well as the non-compiled source code. Thoughts? I'm not super familiar with this action setup, just looking through the docs linked above.

Reply:
You're right, this shouldn't apply to our package. I'll remove this step.